cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java

```diff
@@ -3238,7 +3237,7 @@ private void testSplitterToJoiner(Engine engine) throws Exception {
      * signups --> namesplitter --|
      *                            |non-null --> sink2
      */
-    ETLBatchConfig config = ETLBatchConfig.builder("* * * * *")
```

Comment:
hard to see from the diff: did you keep the schedule for a few of the test cases?

Reply:
not in these tests that run actual pipelines, since none of these tests use a time schedule. It is still used in some other tests that test planning and upgrade.